### ISSUE TITLE
Typo in -l log level help, Quite --> Quiet

### ISSUE THREAD

**Issue by peterjc** [enasequence/sequencetools#8]
```
$ java -jar embl-api-validator-1.1.129.jar
Usage: ena_validator <files> [options]
 
  Options:    -assembly        genome assembly entries (default: false)
    -db              Database connection required for some checks and fixes.Valid databases : ENAPRO or ENADEV. Can only be used within EBI. (optional)(default :false)
    -de              Additional Fix :Adds/Fixes DE line(optional) (default: false)
    -f               File format(optional) Values:'embl','genbank','gff3','assembly' (default: embl)
    -filter          -filter <prefix> Store entries in <prefix>_good.txt and <prefix>_bad.txt files in the working directory. Entries with errors are stored in the bad file and entries without errors are stored in the good file. (optional)(default :false)
    -fix             Fixes entries in input files. Stores input files in 'original_files' folder. (optional) (default: false)
    -fix_diagnose    Creates 'diagnose' folder in the current directory with original entries in <filename>_origin file and the fixed entries in <filename>_fixed file. Only fixed entries will be stored in these files.(optional)  (default: false)
    -help            Displays available options (default: false)
    -l               Log level(optional) Values : 0(Quite),1(Summary),2(Verbose) (default: 1)
    -lowmemory       Runs in low memory usage mode. Writes error logs but does not show message summary(optional) (default: false)
    -min_gap_length  minimum gap length to generate assembly_gap/gap features, use assembly flag to add assembly_gap features (default: 0)
    -prefix          Adds prefix to report files
    -r               Remote, is this being run outside the EBI(optional) (default: false)
    -skip            -skip <errorcode1>,<errorcode2>,... Ignore specified errors.(optional)(default:false) 
    -version         Displays implementation version of Jar (default: false)
    -wrap            Turns on line wrapping in flat file writing (optional)  (default: false)

```

Typo:

```
    -l               Log level(optional) Values : 0(Quite),1(Summary),2(Verbose) (default: 1)
```

Should be:

```
    -l               Log level(optional) Values : 0(Quiet),1(Summary),2(Verbose) (default: 1)
```

Also I'd put a little more white space here.

(I would have submitted a pull request but grep and the GitHub search failed to find the string which puzzles me)

**Comment by satta** [comment]
This seems to have been addressed in the latest master:
```
    -l               Log level(optional) Values : 0(Quiet), 1(Summary), 2(Verbose) (default: 1)
```

**Comment by peterjc** [comment]
Confirmed, fixed in 87c7590683198943f50ad333264a10f3dfb7e62f which added the missing source code to the repository (see also #7).
